samples/zigbee/shell/prj.conf

```diff
@@ -4,8 +4,16 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-CONFIG_NCS_SAMPLES_DEFAULTS=y
-
+# Options enabled by CONFIG_NCS_SAMPLES_DEFAULTS,
+# put this way to get rid of LOG_MINIMAL
+CONFIG_LOG=y
+CONFIG_ASSERT=y
+CONFIG_ASSERT_NO_COND_INFO=y
+CONFIG_ASSERT_NO_MSG_INFO=y
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_LOG_DEFAULT_LEVEL=1
+
+# Configure serial
```

Comment:
@sebastiandraus , do we need to mention the logging options in the shell sample doc? Currently, we only have Logging as one of the dependencies, but maybe it would be good to add at least one line about this?

Reply:
Maybe we could add one more sentence to the Overview? Just to suggest there's logging enabled and it is a bit different than other samples?
```
The commands are logged with the log module name and timestamps.
```